### PR TITLE
Added chaos monkey script

### PR DESCRIPTION
**Pull request by lenuhc**

### CODE REVIEW

File: data/chaos-monkey.yaml [hunk 1]
```diff
@@ -1 +1,162 @@
+```
```
Comment (lenuhc) [inline data/chaos-monkey.yaml]: Is this supposed to be commented??

File: data/chaos-monkey.yaml [hunk 2]
```diff
@@ -1 +1,162 @@
...
+#
+#simianarmy::conformity_properties::simianarmy_conformity_rule_instance_has_health_check_url_enabled: true
+#simianarmy::conformity_properties::simianarmy_conformity_rule_instance_has_status_url_enabled: true
+#simianarmy::conformity_properties::simianarmy_conformity_rule_instance_is_healthy_in_eureka_enabled: true
+#simianarmy::conformity_properties::simianarmy_conformity_rule_instance_too_old_enabled: true
+#simianarmy::conformity_properties::simianarmy_conformity_rule_instance_too_old_instance_age_threshold: 180
+
+
+
+```
```
Comment (michaeldiender-shinesolutions) [inline data/chaos-monkey.yaml]: remove the last line.

File: data/chaos-monkey.yaml [hunk 3]
```diff
@@ -1 +1,162 @@
...
+# simianarmy.chaos.<groupType>.<groupName>.notification.enabled = true
+
+# Set the destination email the termination notification sent to for a specific instance group
+# simianarmy.chaos.<groupType>.<groupName>.ownerEmail = foo@bar.com
+
+# Set the source email that sends the termination notification
+# simianarmy.chaos.notification.sourceEmail = foo@bar.com
+
+
+# janitor.properties
```
Comment (michaeldiender-shinesolutions) [inline data/chaos-monkey.yaml]: add a todo comment saying janitor and conformity still to be implemented.

File: data/chaos-monkey.yaml [hunk 4]
```diff
@@ -1 +1,162 @@
...
+simianarmy::chaos_properties::simianarmy_chaos_asg_max_terminations_per_day: 3.0
+simianarmy::chaos_properties::simianarmy_chaos_leashed: false
+simianarmy::chaos_properties::simianarmy_chaos_asg_probability: 120.0
+simianarmy::chaos_properties::simianarmy_chaos_mandatory_termination_default_probability: 0.5
+simianarmy::chaos_properties::simianarmy_chaos_mandatory_termination_enabled: false
+simianarmy::chaos_properties::simianarmy_chaos_mandatory_termination_window_in_days: 32
+
+
+
+# Five ASG - orchestrator, chaos-monkey, publish, publish-dispatcher, author-dispatcher.
```
Comment (michaeldiender-shinesolutions) [inline data/chaos-monkey.yaml]: this asg properties need to be removed and replaced with puppet task

File: data/chaos-monkey.yaml [hunk 5]
```diff
@@ -1 +1,153 @@
 ---
+
+# simianarmy.properties
+# https://github.com/Netflix/SimianArmy/wiki/Global-Settings
+simianarmy::simianarmy_properties::simianarmy_calendar_close_hour: '15'
```
Comment (michaeldiender-shinesolutions) [inline data/chaos-monkey.yaml]: add simianarmy::client_properties::simianarmy_recorder_sdb_domain: “SIMIAN_ARMY_%{::stack_prefix}"

**Pull request merged**